lib/perl/Genome/Sys.pm

```diff
@@ -1409,6 +1409,12 @@ sub shellcmd {
 
     my ($t1,$t2,$elapsed);
 
+    my @cmdline;
+    if (ref($cmd) and ref($cmd) eq 'ARRAY') {
+        @cmdline = @$cmd;
+        $cmd = join(' ', map $self->quote_for_shell($_), @cmdline));
```

Comment:
It's sorta formal but I think you should rebase or merge `master` to get your `quote_for_shell` change in to this branch.